### PR TITLE
Remove sourceBasePath from commands.yaml and sourcePath from command_sets.yaml

### PR DESCRIPTION
**Pull request by v-shant**
the changes worked in my local i used `github:v-shant/command-sets/remove_source_base_path` as command set location

image

### CODE REVIEW

**Review by niks3089 — COMMENTED** [review]
Is this PR only to remove sourcePath in `command_sets.yaml`? Are you planning to remove sourcePath in `command.yml` next? If so can you edit the PR title to reflect this?

**Comment by v-shant** [comment]
command.yaml can be structured in 2 ways. eg: file.js or folder/file.js so don't know to remove sourcePath from commands.yaml will be beneficial or not.

File: aws/commands.yaml [hunk 1]
```diff
@@ -1,40 +1,39 @@
-sourceBasePath: https://raw.githubusercontent.com/nimbella/command-sets/master/aws/
 commands:
   ec2_list:
-    sourcePath: ec2/ec2_list.js
+    sourcePath: /ec2/ec2_list.js
```
Comment (niks3089) [inline aws/commands.yaml:3]: Move the files under `src`

File: aws/commands.yaml [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
       - name: id
   ec2_status:
-    sourcePath: ec2/ec2_status.js
+    sourcePath: /ec2/ec2_status.js
     description: Know the status of your EC2 instances.
     parameters:
       - name: id
   dynamodb_list:
-    sourcePath: dynamodb/dynamodb_list.js
+    sourcePath: /dynamodb/dynamodb_list.js
```
Comment (niks3089) [inline aws/commands.yaml:19]: Same for this as well

**Review by niks3089 — COMMENTED** [review]
> command.yaml can be structured in 2 ways. eg: file.js or folder/file.js so don't know to remove sourcePath from commands.yaml will be beneficial or not.

Assume the command-sets will be a nimbella project and will have the files under `src` folder. If you make that assumption, we can remove sourcePath in commands.yml as well.

**Comment by v-shant** [comment]
> Assume the command-sets will be a nimbella project and will have the files under src folder. If you make that assumption, we can remove sourcePath in commands.yml as well.

but nimbella project support file name as well. so we will required to check if command.js is present if not then go to command/src/index.js.

are we planning to support only action/src/index.js structure, will not support action.js?

**Review by niks3089 — APPROVED** [review]
LGTM. This is an intermediate PR for the transition between current to nimbella-project. The changes will be merged to dev branch and will not affect the production command-sets.

**Pull request merged**